### ISSUE TITLE
Missing 1.9.1 and 1.9.2 releases

### ISSUE THREAD

**Issue by jishnub** [JuliaLang/Pkg.jl#3554]
Some of the recent changes are not reflected in the documentation, as the releases haven't been tagged

**Comment by jishnub** [comment]
Thanks! Perhaps v1.9.3 should be tagged as well?

**Comment by KristofferC** [comment]
I think 1.9.3 and 1.9.2 Pkg are identical but I guess there could be a tag for 1.9.3 as well.

**Comment by jishnub** [comment]
Indeed, I don't see any Pkg bump in https://github.com/JuliaLang/julia/compare/v1.9.2...v1.9.3, so adding another tag should be enough. This is mainly to be consistent with `Base` and avoid confusing users.
